src/language.rs

```diff
@@ -0,0 +1,89 @@
+extern crate strum;
+
+use std::fmt;
+
+#[derive(PartialEq, Eq, Hash, Clone, EnumString)]
+#[strum(serialize_all = "lowercase")]
```

Comment:
i.e: `The attribute `strum` is currently unknown to the compiler and may have meaning added to it in the future. for more information, see https://github.com/rust-lang/rust/issues/29642`